roles/cs.redis/tasks/install.yml

```diff
@@ -1,4 +1,13 @@
-- name: Install redis packages
+- name: Uninstall redis packages
+  package:
+    name: >-
+      {{ 'valkey' if redis_valkey_flavor == 'redis' else 'redis' }}
+    state: "absent"
+  notify:
+    - Reload systemctl daemon
+  ignore_errors: yes
```

Comment:
why do we need to reload systemctl here?
why ignore_errors? when uninstall fails we might end up with both implementation installed